src/main/java/io/camunda/zeebe/bpmnassert/testengine/ZeebeEngine.java

```diff
@@ -0,0 +1,20 @@
+package io.camunda.zeebe.bpmnassert.testengine;
+
+import io.camunda.zeebe.client.ZeebeClient;
+
+public interface ZeebeEngine extends RecordStreamSource {
+
+  void start();
+
+  void stop();
```

Comment:
:thought_balloon: One thing I pondered is whether we want to add a `reset()` method in the future. This method could clear the in memory database and in memory log storage.

Especially when we think about running this in a test container, we would gain test execution speed if after each test we just do a reset instead of restarting the whole test container.